docs/content/how-to/kubevirt/configuring-network.md

```diff
@@ -0,0 +1,33 @@
+# Configuring Network
+
+By default, the NodePools for the KubeVirt platform generate VMs which are
+attached to the default pod network. It is possible to attach additional
+networks to the NodePools VMs as well though.
+
+## Attaching Additional Networks to KubeVirt VMs
+
+Attaching additional networks to a KubeVirt NodePool is accomplished through
+the usage of multus and [NetworkAttachmentDefinitions](https://docs.openshift.com/container-platform/4.14/rest_api/network_apis/networkattachmentdefinition-k8s-cni-cncf-io-v1.html). Below is an example
```

Comment:
We can add the kubevirt blog practical examples that covers all the aspects not only the nad:
- using ovn multi-homing -> https://kubevirt.io/2023/OVN-kubernetes-secondary-networks.html
- using bridge cni -> https://kubevirt.io/2020/Multiple-Network-Attachments-with-bridge-CNI.html

Reply:
is there an openshift related doc i can link to that covers this?

Reply:
I think this would be the least bad option https://70729--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/multiple_networks/configuring-additional-network